src/Resources/Fulfillment.php

```diff
@@ -14,7 +14,7 @@
 use NVuln\TiktokShop\Resource;
 use SplFileInfo;
 
-class Fulfillment extends Resource
+class Fulfillment extends Resourcenvuln/tiktokshop-php
```

Comment:
@kamil-fityka maybe something wrong here

Reply:
Thanks for your patience. I missed you message and this mistake also.

Reply:
no problem, thanks for your contribution 👍